tensorflow/python/ops/math_ops.py

```diff
@@ -4003,3 +4003,37 @@ def polyval(coeffs, x, name=None):
     for c in coeffs[1:]:
       p = c + p * x
     return p
+
+@tf_export("math.reciprocal_no_nan", "reciprocal_no_nan")
```

Comment:
You need to regenerate the API goldens

Reply:
I don't think I understand. Did you mean API tokens? if so, how do I go about doing that?

Reply:
I found these two commands:
```
    $ bazel build tensorflow/tools/api/tests:api_compatibility_test
    $ bazel-bin/tensorflow/tools/api/tests/api_compatibility_test \
          --update_goldens True
```

Reply:
Can you export just the tf.math version, not the tf.reciprocal_no_nan version?

Reply:
Done in aa5e1db